### PR TITLE
UX: Truncate long category names in sidebar

### PR DESCRIPTION
**Pull request by tgxworld**
### Before 

Screenshot from 2022-06-22 15-23-55

### After

Screenshot from 2022-06-22 15-23-46

### CODE REVIEW

**Comment by jjaffeux** [comment]
After doesn't seem to be vertically aligned

**Comment by tgxworld** [comment]
O good catch should be fixed now

File: app/assets/stylesheets/common/base/sidebar.scss [hunk 1]
```diff
@@ -104,8 +102,24 @@
...
+      width: 70%;
+
+      .badge-wrapper {
+        font-size: 100%;
+        width: 100%;
+
+        .category-name {
+          white-space: nowrap;
+          overflow: hidden;
+          text-overflow: ellipsis;
```
Comment (CvX) [inline app/assets/stylesheets/common/base/sidebar.scss:120]: btw. we have a mixin for this (https://github.com/discourse/discourse/blob/main/app/assets/stylesheets/common/foundation/mixins.scss#L60-L60) but nobody remembers it (incl. usually me 😛) so I kinda question its value 😅

**Review by CvX — APPROVED** [review]

**Comment by tgxworld** [comment]
Thank you for reviewing @jjaffeux and @CvX

**Pull request merged**